assets/src/apps/authoring/store/page/actions/initializeFromContext.ts

```diff
@@ -31,6 +31,9 @@ export const initializeFromContext = createAsyncThunk(
       customCss: params.context.content.customCss,
       custom: params.context.content.custom,
```

Comment:
```suggestion
      custom: params.context.content.custom || {},
```
without this, it gives me an error and fails